pageserver/src/tenant/storage_layer/remote_layer.rs

```diff
@@ -39,6 +46,16 @@ pub struct RemoteLayer {
     pub(crate) ongoing_download: Arc<tokio::sync::Semaphore>,
 }
 
+impl std::fmt::Debug for RemoteLayer {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("RemoteLayer")
+            .field("file_name", &self.file_name)
+            .field("layer_metadata", &self.layer_metadata)
+            .field("is_incremental", &self.is_incremental)
+            .finish()
```

Comment:
would like to see `is_delta` in here.

Reply:
AFAIK It's in the filename (lsn range) -- I was quite close to axing the extra fields in RemoteLayer but just left them out here. Or is there a case where ... we have a single lsn delta layer? Or some other case I couldn't figure out?

Semi-related: While adding the tests to layermap, noted it considers any (delta or image) covering the whole range as `l0_delta` (so lsn is not checked at all). This most likely follows the inability to create an image for anything at 0 or 0xff..ff but still.

Reply:
`is_delta` is not needed because it is always `matches!(self.filename, LayerFileName::Delta(_))` and `file_name` is already included. Should be removed.

Reply:
Not merging until this is clarified :)

Reply:
I think you're right, I was just skimming over the definition of `RemoteLayer` and thought `is_delta` would be relevant information, without checking what it actually means.
Feel free to resolve this conversation.

Reply:
Oki, will type up a tech debt issue of this, resolving and merging.